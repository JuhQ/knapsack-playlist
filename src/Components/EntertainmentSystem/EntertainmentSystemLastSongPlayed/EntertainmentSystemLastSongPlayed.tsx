import React from "react";

const EntertainmentSystemLastSongPlayed: React.FC = () => (
  <h2 className="done">
    Playlist has been played, please generate a new playlist.
  </h2>
);

export default EntertainmentSystemLastSongPlayed;
