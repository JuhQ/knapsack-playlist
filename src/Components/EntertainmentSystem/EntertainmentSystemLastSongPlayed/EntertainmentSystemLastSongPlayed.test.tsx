import React from "react";
import renderer from "react-test-renderer";

import EntertainmentSystemLastSongPlayed from "./EntertainmentSystemLastSongPlayed";

describe("Playlist", () => {
  it("should render", () => {
    const component = renderer.create(<EntertainmentSystemLastSongPlayed />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
