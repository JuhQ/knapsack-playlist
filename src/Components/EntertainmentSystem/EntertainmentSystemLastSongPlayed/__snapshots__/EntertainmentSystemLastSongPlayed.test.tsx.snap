// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Playlist should render 1`] = `
<h2
  className="done"
>
  Playlist has been played, please generate a new playlist.
</h2>
`;
